store/sqlstore/license_store.go

```diff
@@ -66,13 +68,13 @@ func (ls SqlLicenseStore) Save(license *model.LicenseRecord) (*model.LicenseReco
 // Get obtains the license with the provided id parameter from the database.
 // If the license doesn't exist it returns a model.AppError with
 // http.StatusNotFound in the StatusCode field.
-func (ls SqlLicenseStore) Get(id string) (*model.LicenseRecord, *model.AppError) {
+func (ls SqlLicenseStore) Get(id string) (*model.LicenseRecord, error) {
 	obj, err := ls.GetReplica().Get(model.LicenseRecord{}, id)
 	if err != nil {
-		return nil, model.NewAppError("SqlLicenseStore.Get", "store.sql_license.get.app_error", nil, "license_id="+id+", "+err.Error(), http.StatusInternalServerError)
+		return nil, errors.Wrapf(err, "failed to get License with licenseId=%s", id)
 	}
 	if obj == nil {
-		return nil, model.NewAppError("SqlLicenseStore.Get", "store.sql_license.get.missing.app_error", nil, "license_id="+id, http.StatusNotFound)
+		return nil, store.NewErrNotFound("License", fmt.Sprintf("licenceId=%s", id))
```

Comment:
Is it necessary to pass `licenseId=`? The error already assumes that it's an id, and prints `id: `.

Reply:
you are right, I forgot that